### ISSUE TITLE
New maintainer!

### ISSUE THREAD

**Issue by joeced** [sphinx-contrib/matlabdomain#266]
I've authored and maintained `sphninxcontrib-matlabdomain` for the past 7 years. It's been exiting, challenging, but very rewarding to use it at work and see other people and projects using it. I've learned a ton of things a long they way.

However, I'm starting a new job on October 1st 2024, which does not involve MATLAB at all.
I will therefore have no chance of checking if MATLAB code is valid. I can still work a little on the project on the side, but probably way less than before.

I'm therefore looking for someone to take over the role as maintainer / main author. If you are interested, then please contact me here or by email (you should be able to find it via my GitHub profile).

Tagging: @rdzman, @apozharski.

**Comment by borlum** [comment]
Tak for god software, @joeced! 💯 
Hoping someone will pick-up the maintainer role to carry on - we are a bunch of silent but happy users 😄

**Comment by Remi-Gau** [comment]
My future bandwidth may regret this but: given how much I use this extension I'd like to help with the maintenance though I am not really familiar with the codebase.

Would prefer if I were not alone to avoid a bus factor of 1.

**Comment by joeced** [comment]
THANKS A LOT! 

That went much more smooth than I feared. I've given @apozharski and @Remi-Gau as "Maintainter" status on the project. You should now be able to approve pull requests and merge to master.

I'm deeply thankful for new people taking over this project!

**Comment by joeced** [comment]
@Remi-Gau  and @apozharski, in order to add you to the Read-the-docs project, you should login or signup at https://about.readthedocs.com/. I just used the "Login with GitHub" option in the past. Once that it is done, I can add you to the project. Alternatively, I can add you by email.

**Comment by Remi-Gau** [comment]
I am logged but could not see [sphinxcontrib-matlabdomain](https://readthedocs.org/projects/sphinxcontrib-matlabdomain/) in my list of projects

in case that helps my email is remi_gau@hotmail.com

**Comment by apozharski** [comment]
Yep I have an account attached to my github on RTD, so adding that way should be possible.
If easier, email: apozharski@gmail.com

**Comment by joeced** [comment]
Added to RTD:
![image](https://github.com/user-attachments/assets/6bf1a5a5-9125-40a4-9c3a-31ca86c40b71)

**Comment by Remi-Gau** [comment]
Merci !!!!

**Comment by rdzman** [comment]
Great news ... I'm happy to contribute where I can, but there was no way I was going to be an adequate maintainer.

**Comment by apozharski** [comment]
One thing that has fallen through for a while for me is pushing to completion the parsing cleanup that I started a while ago using the Tree-Sitter based parser (#261) . I have been unexpectedly working on a non-matlab codebase for the last months so I haven't taken too much time here. However in late July I will have once again time to focus on this.

@jasonnicholson You had previously full class support in "wants". Is there something still missing in the latest development head that you would still want. If there is I suspect it would not be too difficult to add, and in the worst case would require some contributions to [tree-sitter-matlab](https://github.com/acristoffers/tree-sitter-matlab) if there are some parsing missing.

**Comment by jasonnicholson** [comment]
There are several features I'm looking for. 

- The first one would be parsing size and types from properties and arguments blocks when they are defined. This would be a feature. 
- I'm looking to add support for the Matlab markup format. 
- I am looking for a better way define a code block in the docstring. >>> Is to pythonic. :: is what I am using.
- some docstrings first line render as a heading for no foreseeable reason. No indent after. No heading keyword. It's weird. 


I may have more later. I am heading out for a run with the dogs.
